lib/improver/psychrometric_calculations/psychrometric_calculations.py

```diff
@@ -802,13 +802,17 @@ def process(self, temperature, relative_humidity, pressure, orog,
             snow_cube.data = self.find_falling_level(wb_integral.data,
                                                      orog_data,
                                                      height_points)
-            # Interpolate missing data
-            snow_cube.data = self.fill_in_missing_data(snow_cube.data,
-                                                       orog_data,
-                                                       land_sea_data,
-                                                       wb_integral.data[0, ::],
-                                                       highest_height)
-
+            # Fill in missing data
```

Comment:
In the process doc-string (way above this line), "corresponding" has an interesting spelling.

Reply:
Fixed